osfoffline/views/start_screen.py

```diff
@@ -1,12 +1,22 @@
+import furl
+import aiohttp
```

Comment:
I find triangular import organization more visually appealing (each import line per section is longer than the previous), but can change it to alphabetical if it's very important to you.